service/utils/tds.py

```diff
@@ -318,3 +319,22 @@ def attach_files(output: dict, job_id, status="complete"):
         job_id, status=status, result_files=list(files.values()), finish=True
     )
     logging.info("uploaded files to %s", job_id)
+
+
+def fetch_interventions(policy_intervention_id: str, job_id):
+    job_dir = get_job_dir(job_id)
+    logging.debug(f"Fetching interventions {policy_intervention_id}")
+
+    intervention_url = TDS_URL + TDS_INTERVENTIONS + "/" + policy_intervention_id
+    intervention_response = tds_session().get(intervention_url)
+    if intervention_response.status_code == 404:
+        raise HTTPException(status_code=404, detail="Intervention not found")
+
+    intervention_path = os.path.join(job_dir, f"./{policy_intervention_id}.json")
+    with open(intervention_path, "w") as file:
+        # Ensure we don't have null observables which can be problematic downstream, if so convert
+        # to empty list
+        intervention_json = intervention_response.json()
+        json.dump(intervention_json, file)
```

Comment:
- What is the purpose of dumping the content into a file? 

- I don't understand the observables comment, you are also not doing anything with observables here

Reply:
obs comment is leftover from copy paste
I figured dumping into a file was okay for debugging - similar to how we are dumping AMRs into files as well.
If you think we should not I can remove that

Reply:
Debugging file is fine.